compiler/src/dotty/tools/dotc/core/TypeComparer.scala

```diff
@@ -2838,7 +2838,7 @@ class TypeComparer(@constructorOnly initctx: Context) extends ConstraintHandling
           else
             false
       case (AppliedType(tycon1, args1), AppliedType(tycon2, args2)) if isSame(tycon1, tycon2) =>
-        // It is possible to conclude that two types applies are disjoint by
+        // It is possible to conclude that two types applied are disjoint by
         // looking at covariant type parameters if the said type parameters
         // are disjoin and correspond to fields.
```

Comment:
disjoint

Reply:
eh?

Reply:
`// are disjoin and correspond to fields.`

Reply:
oh. true, but this PR doesn't fix every typo 🤷